### DISCUSSION TITLE
Ryzen 9 5950 X

### DISCUSSION THREAD

**Discussion by hmercker** [catchthatrabbit/coreminer#58] [Profitability]
Ryzen 9 5950 X

CPU: 16 cores, 32 threads, 3.6 GHz base clock, Ryzen 9 5950X
RAM: 16 Gb Ripjaw
GPU: Geforce GTX 710
motherboard: MSI B550 - Tomahawk
Cooler: Dark Rock Pro 4 - be quiet!
PSU: Corsair 750 W powersupply

Hash rate 19-20 kH/s

**Comment by TobiW47** [comment]
Ryzen 9 3950x, 16 threads, x470, 16GB DDR4 3200MHz makes consistently 18.4 KH/sec
